.ci/Jenkinsfile.gosh-prepare-rc

```diff
@@ -158,7 +158,7 @@ pipeline {
                         } else {
                             sh """      
                                 aws s3api create-bucket --bucket ${BUCKET_NAME} --region eu-west-2 --acl public-read --create-bucket-configuration LocationConstraint=eu-west-2
-                                aws s3 website s3://${BUCKET_NAME} --index-document index.html --error-document error.html
+                                aws s3 website s3://${BUCKET_NAME} --index-document index.html --error-document index.html
```

Comment:
Why do we replace error document here?

Reply:
This change appeared after merging dev to this branch